aws/table_aws_inspector_finding.go

```diff
@@ -0,0 +1,364 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/service/inspector"
+	"github.com/turbot/go-kit/helpers"
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
```

Comment:
Is there a reason we use an alias here, since we don't use them in other plugin tables? Is there an advantage to it, or helps resolve a conflict?